test/e2e/workflow/deployment_test.go

```diff
@@ -128,6 +144,11 @@ var _ = Describe("NetworkAddonsConfig", func() {
 			configSpec.MacvtapCni = &cnao.MacvtapCni{}
 			components = append(components, MacvtapComponent)
 			testConfigUpdate(gvk, configSpec, components)
+
+			// Add Multus Dynamic Networks component (requires multus ...)
+			configSpec.Multus = &cnao.Multus{}
+			configSpec.MultusDynamicNetworks = &cnao.MultusDynamicNetworks{}
+			components = append(components, MultusComponent, MultusDynamicNetworks)
```

Comment:
This does not take any effect unless you call

```
testConfigUpdate(gvk, configSpec, components)
```

Reply:
Done.